plugins/VisitFrequency/Reports/Get.php

```diff
@@ -32,14 +32,25 @@ protected function init()
         $this->processedMetrics = array(
             new ReturningMetric(new AverageTimeOnSite()),
             new ReturningMetric(new ActionsPerVisit()),
-            new ReturningMetric(new BounceRate())
+            new ReturningMetric(new BounceRate()),
+            new ReturningMetric(new AverageTimeOnSite(), '_new'),
+            new ReturningMetric(new ActionsPerVisit(), '_new'),
+            new ReturningMetric(new BounceRate(), '_new')
         );
         $this->metrics       = array(
             'nb_visits_returning',
             'nb_actions_returning',
             'nb_uniq_visitors_returning',
             'nb_users_returning',
-            'max_actions_returning'
+            'max_actions_returning',
+            'nb_visits_new',
+            'nb_actions_new',
+            'nb_uniq_visitors_new',
+            'nb_users_new',
+            'max_actions_new',
+            'bounce_rate_new',
+            'nb_actions_per_visit_new',
+            'avg_time_on_site_new'
```

Comment:
@katebutler I reckon we don't need to add columns like `nb_actions_per_visit_new` etc but only need to add the same columns that are there for returning?